src/main/kotlin/com/example/noswimmingserver/domain/reading_journal/presentation/ReadingJournalController.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.noswimmingserver.domain.reading_journal.presentation
+
+import com.example.noswimmingserver.domain.reading_journal.presentation.dto.request.CreateReadingJournalRequest
+import com.example.noswimmingserver.domain.reading_journal.service.CreateReadingJournalService
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)